protocol/0038-OLIQ-liquidity_provision_order_type.md

```diff
@@ -89,21 +99,15 @@ The sum of all normalised proportions must = 1 for all refined buy / sell order
 
 #### Calculating volumes for a set of market making orders (step 6):
 
-From the network parameter `market.liquidity.minimum.probabilityOfTrading.lpOrders` and from `best static bid-price` we get `minPrice` from the [Quant risk model spec](./0018-RSKM-quant_risk_models.ipynb): the smallest price level that has probability of trading greater than or equal to `market.liquidity.minimum.probabilityOfTrading.lpOrders`. 
-Similarly from `best static ask-price` we get `maxPrice`: the largest price level that has probability of trading greater than or equal to `market.liquidity.minimum.probabilityOfTrading.lpOrders`. 
-Any shape entry with a peg less than `minPrice` should have the resulting volume implied at `minPrice` (instead of what the level the peg would be) while any shape entry with peg greater than `maxPrice` should have the resulting volume implied at `maxPrice`. 
+Any shape entry with a peg less than `min_lp_vol_price` should have the resulting volume implied at `min_lp_vol_price` (instead of what the level the peg would be) while any shape entry with peg greater than `max_lp_vol_price` should have the resulting volume implied at `max_lp_vol_price`. 
```

Comment:
```suggestion
Any shape entry with a peg less than `min_lp_vol_price` should have the resulting volume implied at `min_lp_vol_price` (instead of what level the peg would be) while any shape entry with peg greater than `max_lp_vol_price` should have the resulting volume implied at `max_lp_vol_price`. 
```